frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -160,7 +269,25 @@ export const ProfilePage: React.FC<ProfilePageProps> = ({
                   description: "Title of the My hiring pools section",
                 })}
               </h2>
-              <p>Pool details</p>
+              <div
+                data-h2-bg-color="b(gray)"
+                data-h2-padding="b(all, m)"
+                data-h2-radius="b(s)"
+              >
+                {(candidateArray === null || candidateArray.length === 0) && (
```

Comment:
You're doing a lot of `=== null` and `=== undefined` and `=== 0` in this file. These are all considered falsy in javascript.  You could using the more concise form of direct boolean evalution like `!candidateArray || !candidateArray.length` to shorten the code.  I see you doing this in other places on this file so I suggest that form to be consisent.